scripts/analytics/benchmarks.py

```diff
@@ -138,7 +139,101 @@ def get_log_counts(users):
     return rows
 
 
+def get_projects():
+    projects = Node.find(
+        Q('category', 'eq', 'project') &
+        Q('is_deleted', 'eq', False) &
+        Q('is_folder', 'ne', True)
+    )
+    return projects
+
+def get_projects_forked():
+    projects_forked = list(Node.find(
+        Q('category', 'eq', 'project') &
+        Q('is_deleted', 'eq', False) &
+        Q('is_folder', 'ne', True) &
+        Q('is_fork', 'eq', True)
+    ))
+    return projects_forked
+
+def get_projects_registered():
+    projects_registered = Node.find(
+        Q('category', 'eq', 'project') &
+        Q('is_deleted', 'eq', False) &
+        Q('is_folder', 'ne', True) &
+        Q('is_registration', 'eq', True)
+    )
+    return projects_registered
+
+def get_projects_public():
+    projects_public = Node.find(
+        Q('category', 'eq', 'project') &
+        Q('is_deleted', 'eq', False) &
+        Q('is_folder', 'ne', True) &
+        Q('is_public', 'eq', True)
+    )
+    return projects_public
+
+
+def get_number_downloads_unique_and_total():
+    number_downloads_unique = 0
+    number_downloads_total = 0
+
+    projects = get_projects()
+
+    contributors_per_project = []
```

Comment:
This method returns number of downloads of unique and total but this variable is not used in return, it is not necessary.